radio/src/targets/common/arm/stm32/adc_driver.cpp

```diff
@@ -24,10 +24,12 @@
   // not needed
 #elif defined(PCBX10)
   const int8_t ana_direction[NUM_ANALOGS] = {1,-1,1,-1,  -1,-1,-1, -1,1,1,1,  -1};
-#elif defined(PCBX9E) && defined(HORUS_STICKS)
-  const int8_t ana_direction[NUM_ANALOGS] = {1,-1,1,-1,  -1,-1,-1,1, -1,1,1,1,  -1};
 #elif defined(PCBX9E)
-  const int8_t ana_direction[NUM_ANALOGS] = {1,1,-1,-1,  -1,-1,-1,1, -1,1,1,1,  -1};
+  #if defined(HORUS_STICKS)
```

Comment:
We indent ifdef in a few places. I felt here it improved readability but I can change it back to no indentation. I have no strong feelings either way.